File: src/main/java/jp/osak/haggledehaghag/config/ApplicationConfig.java
package jp.osak.haggledehaghag.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jdbc.repository.config.EnableJdbcRepositories;

@Configuration
@EnableJdbcRepositories("jp.osak.haggledehaghag.repository")
public class ApplicationConfig {
}

File: src/main/java/jp/osak/haggledehaghag/config/ApplicationConfig.kt
package jp.osak.haggledehaghag.config

import org.springframework.context.annotation.Configuration
import org.springframework.data.jdbc.repository.config.EnableJdbcRepositories
import org.springframework.transaction.annotation.EnableTransactionManagement

@Configuration
@EnableJdbcRepositories("jp.osak.haggledehaghag.repository")
@EnableTransactionManagement
open class ApplicationConfig